docs/src/_includes/components/docs-index.html

```diff
@@ -10,16 +10,42 @@
 </li>
 {%- endmacro %}
 
-<nav class="docs-index" id="docs-index">
-    <button class="docs-index-toggle" aria-label="Index" id="js-docs-index-toggle" hidden>
+<nav class="docs-index" id="docs-index" >
+    <button class="docs-index-toggle" aria-label="Index" id="js-docs-index-toggle">
         Index
         <svg width="20" height="20" viewBox="20 20 60 60">
             <path id="ham-top" d="M30,37 L70,37 Z" stroke="currentColor"></path>
             <path id="ham-middle" d="M30,50 L70,50 Z" stroke="currentColor"></path>
             <path id="ham-bottom" d="M30,63 L70,63 Z" stroke="currentColor"></path>
         </svg>
     </button>
+    <noscript>
+        <style>
+            #js-docs-index-toggle{
+                display: none;
+            }
+        </style>
+        <button class="docs-index-toggle" aria-label="Index"  hidden>
+            Index
+            <svg width="20" height="20" viewBox="20 20 60 60">
+                <path id="ham-top" d="M30,37 L70,37 Z" stroke="currentColor"></path>
+                <path id="ham-middle" d="M30,50 L70,50 Z" stroke="currentColor"></path>
+                <path id="ham-bottom" d="M30,63 L70,63 Z" stroke="currentColor"></path>
+            </svg>
+        </button>
```

Comment:
This looks like a dead code. It's hidden, and since it is in `<noscript>` there will be no script run to unhide it.

Reply:
yes. We don't need this.